infrahub_sdk/ctl/cli_commands.py

```diff
@@ -187,19 +195,34 @@ def render_jinja2_template(template_path: Path, variables: dict[str, str], data:
 
 
 def _run_transform(
-    query: str,
+    query_name: str,
+    client: InfrahubClient,
```

Comment:
Does it makes sense to pass an InfrahubClient instance into this method? The only reason we seem to need this is for the graphql query. If we use the `run` method of the transform instead, then it will handle querying the data for us.

The type you are using here is the async version of the client, but this is a sync function, and we are calling a sync method on the `client`, so it should probably be `InfrahubClientSync`. When you are calling this function, you seem to be passing in the sync version of the client.

Reply:
I ended up updating the transform so it tries the stored query API endpoint first then tries the graphQL API directly afterwards.